bundles/core/org.eclipse.smarthome.core.audio.test/src/test/groovy/org/eclipse/smarthome/core/audio/test/AudioManagerTest.groovy

```diff
@@ -36,10 +36,17 @@ public class AudioManagerTest extends AudioOSGiTest {
         assertProcessedStream(audioStream, PlayType.STREAM)
     }
 
-    @Test(expected = NullPointerException.class)
+    @Test
     public void 'null streams are not processed'(){
         registerSink()
+
         audioManager.play(null, audioSinkFake.getId())
+
+        waitForAssert({    
+            assertThat "null stream was processed",    
+                    audioSinkFake.isStreamProcessed,
+                    is(false)  
```

Comment:
The JavaDoc says that `null` streams should stop the current running stream. Isn't this also a sort of processing? Maybe the `AudioSinkFake` should indicate that the fake stream was stopped and the test should not talk about "not processed". wdyt?

Reply:
Yes, I saw this too. We should indicate whether a processed stream is playing or not.